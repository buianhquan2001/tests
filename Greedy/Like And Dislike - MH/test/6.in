19
18 100 28
17 63 18
74 42 76
38 81 57
44 0 86
81 52 56
75 73 40
91 85 67
58 61 81
89 51 31
65 30 92
80 100 77
21 76 87
32 2 93
16 72 8
37 59 21
59 80 5
45 16 62
55 21 36
