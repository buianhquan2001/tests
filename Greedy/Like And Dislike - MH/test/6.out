Dislike
Dislike
Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
Like
Unknown
Unknown
Unknown
Unknown
Dislike
Dislike
Dislike
Unknown
Unknown
